app/server/Main.hs

```diff
@@ -1,8 +1,53 @@
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
 module Main where
 
 import Cardano.Wallet
     ( sayHello )
+import Control.Monad
+    ( when )
 import Prelude
+import System.Console.Docopt
+    ( Docopt
+    , argument
+    , docopt
+    , exitWithUsage
+    , getArgOrExitWith
+    , isPresent
+    , longOption
+    , parseArgsOrExit
+    )
+import System.Environment
+
+
+-- | Command-Line Interface specification. See http://docopt.org/
+cli :: Docopt
+cli = [docopt|
+cardano-wallet-server
+
+Start the cardano wallet along with its API and underlying node.
+
+Usage:
+  cardano-wallet-server [options]
+  cardano-wallet-server --help
+
+Options:
+  --wallet-server-port  Number of core nodes to start [default: 4]
+  --node-port           Number of relay nodes to start [default: 1]
+  --network             mainnet or testnet [default: mainnet]
+|]
```

Comment:
Note that we could use `docoptFile` here and point to a file in `specifications/launcher`

Reply:
Isn't it nice to have it here, in the same place?

Reply:
I added a readme file now at `specifications/CLI.md` (wrong place?)

About symlink: In that case we would have to make sure the module level comment targets both developers and users… which might arguably be a good thing.

I could try that.